### DISCUSSION TITLE
can this be used to run winget upgrade --all as system in SCCM?

### DISCUSSION THREAD

**Discussion by brianmantler** [Romanitho/Winget-AutoUpdate#126] [Q&A]
I am trying to use our config manager environment to auto update third party apps across the enterprise.  I am running into the issue of winget not running as system..

I see your winget-install project runs in SCCM as system, but I don't think you have to specify specific packages, and not --all.

Can this project run as system in SCCM with an --all option?

Thanks

**Comment by Romanitho** [comment]
Hello. Basically this project works under SCCM or Intune. If you want to upgrade all, you have to create an empty black list file and install WAU as described. 
But i don't recommand that as you may want to exclude browsers or office for instance
